### DISCUSSION TITLE
Easier access to installation instructions on landing page

### DISCUSSION THREAD

**Discussion by NicklasXYZ** [gleam-lang/gleam#2703] [Ideas & suggestions]
I have had to install gleam on a couple of different machines over a period of time so I find myself having to navigate to the installation instruction for various operating systems.

I typically find myself having to click/scroll around on the landing page before finding the right link through the "Docs" link in the top nav bar.
Going to the landing page of dlang, rust, elixir, erlang, F#, nim, etc. links to installation instructions and binaries are more easily accessible and prominently placed on the landing page.

**Comment by lpil** [comment]
I'm up for the idea in theory but I don't have the web design skills to be able to achieve this nicely.

**Comment by markholmes** [comment]
How about something like this? Happy to submit a PR:

<img width="1010" alt="Screenshot 2024-03-10 at 1 04 53 PM" src="https://github.com/gleam-lang/website/assets/921826/332be6a9-0919-4257-bd66-fdf36868f539">

**Comment by NicklasXYZ** [comment]
yeah, to me that's excellent :)

**Comment by lpil** [comment]
I don't want to have a second call to action on the hero unless the installation process is as slick as getting started experience as the tour. Getting people to try writing Gleam code as quickly as possible is the goal of the website here.

Making it easier to find the install guides without using the documentation page is not a goal at all. A search engine is a much better option in that case.

**Reply by markholmes** [reply]
Is there a longer term goal to instead focus on improving installation instructions?
